packages/js/manifests/polywrap/src/formats/polywrap.app/0.2.0.ts

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+/* tslint:disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export interface AppManifest {
+  /**
+   * Polywrap manifest format version.
+   */
+  format: "0.2.0";
+  /**
+   * Basic project properties.
+   */
+  project: {
+    /**
+     * Name of this project.
+     */
+    name: string;
+    /**
+     * Type of this project.
+     */
+    type: string;
```

Comment:
I find it weird to have `type` inside the `app` manifest - Not sure tho...

Reply:
If we take a look at the spec https://github.com/polywrap/monorepo/issues/1057 - the `type` field is a common field for all polywrap manifests (plugin, wasm, app, interface).

If I am to run `polywrap app xyz`, how would the CLI be sure that the current project is an app, unless the `type` explicitly said so? 😄

LMK if there's something I missed.